pkg/restapi/issuer/operation/operations_test.go

```diff
@@ -496,6 +496,31 @@ func testCreateProfileHandler(t *testing.T) {
 
 		require.Equal(t, invalidRequestErrMsg+": EOF", errResp.Message)
 	})
+	t.Run("create profile error unable to create vault in edv", func(t *testing.T) {
```

Comment:
Add a `-` or `:` between the "create profile error" and the "unable" in order to make it easier to read

Reply:
fixed!